pandas/tests/io/test_parquet.py

```diff
@@ -724,6 +744,13 @@ def test_timestamp_nanoseconds(self, pa):
         df = pd.DataFrame({"a": pd.date_range("2017-01-01", freq="1n", periods=10)})
         check_round_trip(df, pa, write_kwargs={"version": "2.0"})
 
+    def test_timezone_aware_index(self, pa, timezone_aware_date_list):
+        idx = 5 * [timezone_aware_date_list]
+        df = pd.DataFrame(index=idx, data={"index_as_col": idx})
+
+        # compare time(zone) values only, skip their class
```

Comment:
Can you explain in the comment what is different about the dtype object (different timezone class: pytz.FixedOffset vs datetime.timezone)